index.css

```diff
@@ -688,3 +712,756 @@ form {
+    position: static;
+    height: 20px;
+    left: 12px;
+    top: 10px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    color: #6070ff;
+  }
+
+  .toolbar_class .option_bar {
+    position: absolute;
+    height: 12px;
+    background: #6070ff;
+    right: -100px;
+  }
+
+  .span_class {
+    display: none;
+  }
+
+  .navlist {
+    position: relative;
+    list-style-type: none;
+    left: 400px;
+    display: flex;
+    flex-direction: row;
+    top: -5px;
+  }
+
+  .navlist-item {
+    margin: 0 30px;
+    text-decoration: none;
+  }
+
+  .navlist-item > a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    line-height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+  }
+
+  .background {
+    position: absolute;
+    width: 70%;
+    height: 500px;
+    top: 119px;
+    margin: 1px 100px 1px 100px;
+    left: 100px;
+  }
+
+  .background .intro_class {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 20px;
+    position: absolute;
+    width: 90%;
+    height: 268px;
+    margin-top: 10px;
+  }
+
+  .background .intro_class .identity_text {
+    position: static;
+    width: 70%;
+    height: 50px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 40px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 6px 1px;
+    overflow-wrap: break-word;
+  }
+
+  .background .intro_class .description_text {
+    position: relative;
+    width: 90%;
+    height: 100px;
+    top: 1px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: content;
+    order: 0;
+    flex-grow: 0;
+    margin: 6px 1px;
+    margin-bottom: 10px;
+  }
+
+  .background .intro_class .connect_text {
+    position: relative;
+    width: 95%;
+    height: 70px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    flex: auto;
+    order: 0;
+    flex-grow: 0;
+    margin: 30px 1px;
+  }
+
+  .background .intro_class .social_media_icons {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    left: -46px;
+    top: -30px;
+    flex: auto;
+  }
+
+  .background .intro_class .social_media_icons .icons {
+    margin: 2px 7px;
+    left: -25px;
+  }
+
+  .left_ellipse {
+    position: relative;
+    top: 20px;
+    left: 1px;
+    transform: scale(-0.4, -0.4);
+  }
+
+  .left_rectangle {
+    position: relative;
+    top: 200px;
+    left: 500px;
+    transform: rotate(20deg);
+  }
+
+  .bottom_ellipse {
```

Comment:
- [ ] Please as it was requested from the last reviewer make sure that the background matches the one in Figma design 
![image](https://user-images.githubusercontent.com/74744787/160117362-016016ff-6f85-411b-a5fa-e1d88ee6e893.png)
![image](https://user-images.githubusercontent.com/74744787/160117517-e9bfbcfd-ea6b-47e5-80fe-34be8ce3b1de.png)

I think it would be better if you downloaded the same image as in Figma design